src/navigation/tabs/settings/SettingsRoot.tsx

```diff
@@ -11,49 +12,42 @@ import {
   Setting,
   SettingIcon,
   SettingTitle,
+  Hr,
 } from '../../../components/styled/Containers';
 import {RootState} from '../../../store';
 import {User} from '../../../store/bitpay-id/bitpay-id.models';
-import {Black, Feather, LightBlack, White} from '../../../styles/colors';
-import ChevronDownSvg from '../../../../assets/img/chevron-down.svg';
-import ChevronUpSvg from '../../../../assets/img/chevron-up.svg';
-import {navigationRef} from '../../../Root';
-import {updateSettingsListConfig} from '../../../store/app/app.actions';
-import {useAppSelector, useAppDispatch} from '../../../utils/hooks';
-
-import General from './components/General';
-import Security from './components/Security';
-import Notifications from './components/Notifications';
-import Connections from './components/Connections';
-import ExternalServices from './components/ExternalServices';
-import About from './components/About';
-import Contacts from './components/Contacts';
+import {useAppDispatch} from '../../../utils/hooks';
 import {useSelector} from 'react-redux';
-import Crypto from './components/Crypto';
-import WalletsAndKeys from './components/WalletsAndKeys';
 import {useScrollToTop} from '@react-navigation/native';
 import {SettingsScreens, SettingsGroupParamList} from './SettingsGroup';
 
-interface HomeSetting {
-  id: SettingsListType;
-  title: string;
-  onPress: () => void;
-  subListComponent: ReactElement;
-}
+export type SettingsListType =
+  | 'General'
+  | 'Contacts'
+  | 'Crypto'
+  | 'Wallets & Keys'
+  | 'Security'
+  | 'External Services'
+  | 'Notifications'
+  | 'Connections'
+  | 'About BitPay';
 
-export const SettingsContainer = styled.SafeAreaView`
-  flex: 1;
-`;
+export type SettingsHomeProps = NativeStackScreenProps<
+  SettingsGroupParamList,
+  SettingsScreens.SETTINGS_HOME
+>;
 
-export const Settings = styled.ScrollView`
-  padding: 10px ${ScreenGutter};
+export const SettingsContainer = styled(SafeAreaView)`
```

Comment:
Any reason why we changed the ScrollView to a SafeAreaView? This is causing the listings to stop being scrollable. For example in the "Wallets & Keys" listing, contacts, external services history, languages ​​listing, etc.

Reply:
Good catch. Now fixed.